app/templates/_gradle.properties

```diff
@@ -1,7 +1,8 @@
 rootProject.name=<%= _.slugify(baseName) %>
 profile=dev
-
-assertj_core_version=1.6.1
+<% if (javaVersion == '8') { %>
+assertj_core_version=1.0.0m1<% } else { %>
```

Comment:
Can you do a if (javaVersion == '7') instead of the "else" everywhere? I know this code might stay here for long, and I don't want strange behavior when we use Java 9... I have the same issues with the databases